tendrl/integrations/gluster/utils.py

```diff
@@ -0,0 +1,22 @@
+def update_volume_alert_count(alert, existing_alert):
+    counter_obj = NS.integrations.gluster.objects.VolumeAlertCounters(
+        integration_id=alert.tags['integration_id'],
+        volume_id=alert.tags['volume_id']
+    ).load()
+    warn_count = int(counter_obj.warning_count)
+    info_count = int(counter_obj.info_count)
+    if existing_alert:
+        if alert.severity == "INFO":
```

Comment:
Do you really need this?
I feel just increment or decrement  the alert count.
Also if you remove info, you can call it just alert count.